src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerViewController.swift

```diff
@@ -47,10 +48,11 @@ class QRScannerViewController: UIViewController {
 						// Don't show an error in simulator to enable debugging/UI-Tests
 						return
 						#else
-						self?.showCameraPermissionErrorAlert()
+						self?.viewModel?.deactivateScanning()
```

Comment:
This leads to an infinite loop and crashes the app if the camera permission was denied
```suggestion
```